build-docker.sh

```diff
@@ -20,12 +20,13 @@ container="$(
 )"
 trap "docker rm -f ${container}" exit
 
-docker cp "${TARGET_DIR}/portier-broker" "${container}:/"
+docker cp "${TARGET_DIR}/portier-broker" "${container}:/tmp/portier-broker"
```

Comment:
This looks like it should be folded into e34b1f99750b29f4ced4e85910e2edfdc9a2251c, otherwise that commit has an invalid docker file creation script, right?